docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -383,164 +266,76 @@ commodore catalog compile ${CLUSTER_ID} --push -i
 
 . Configure Terraform secrets
 +
-[source,console]
+[source,bash]
 ----
-cat <<EOF > catalog/manifests/openshift4-cloudscale/.env
+cat <<EOF > ./terraform.env
 CLOUDSCALE_TOKEN
 TF_VAR_ignition_bootstrap
+TF_VAR_lb_cloudscale_api_secret
+TF_VAR_control_vshn_net_token
+GIT_AUTHOR_NAME
+GIT_AUTHOR_EMAIL
+HIERADATA_REPO_TOKEN
 EOF
 ----
 
 include::partial$setup_terraform.adoc[]
 
-. Provision bootstrap node
+. Create LB hieradata
 +
-[source,console]
+[source,bash]
 ----
 cat > override.tf <<EOF
 module "cluster" {
-  bootstrap_count = 1
+  bootstrap_count = 0
   master_count    = 0
   infra_count     = 0
   worker_count    = 0
 }
 EOF
-
 terraform apply
 ----
 
-. Create the first shown DNS records
+. Ensure that all the DNS records shown in output variable `dns_entries` from the previous step are configured in the cluster's parent zone.
```

Comment:
I'm actually not sure if this step is in the right place, this should most likely be moved after the "Create LBs" step.

Reply:
I've moved it to after "create LBs" step. this would also solve the discussion earlier.